s3/replication/replicator/src/s3replicator/replicator_routes.py

```diff
@@ -35,12 +36,26 @@
 @routes.get('/jobs')  # noqa: E302
 async def list_jobs(request):
     """List all in-progress jobs."""
-    _logger.debug('API: GET /jobs')
-    jobs = request.app['all_jobs']
+    url = request.path_qs
+    url_ob = urlparse(url)
+    query = parse_qs(url_ob.query, keep_blank_values=True)
+    _logger.debug('API: GET /jobs\n Query: {}'.format(query))
 
-    _logger.debug('Number of jobs in-progress {}'.format(jobs.count()))
-    # _logger.debug('List of jobs in-progress {}'.format(Jobs.dumps(jobs)))
-    return web.json_response(jobs, dumps=Jobs.dumps, status=200)
+    jobs = request.app['all_jobs']
+    completed_jobs = request.app['completed_jobs']
+
+    if 'count' in query:
+        if 'inprogress' in query:
+            return web.json_response({'In-progress jobs count':
```

Comment:
json keys should be parseable, it can be just `count` in call cases below or `inprogress-count`.
same at below lines, either just `count` or `completed-count` and `count` for all.